src/modules/data/blob.c

```diff
@@ -9,12 +9,28 @@ Blob* lovrBlobCreate(void* data, size_t size, const char* name) {
   blob->data = data;
   blob->size = size;
   blob->name = lovrStrdup(name);
+  blob->root = blob;
+  return blob;
+}
+
+Blob* lovrBlobCreateView(Blob* parent, size_t offset, size_t size, const char* name) {
+  Blob* blob = lovrCalloc(sizeof(Blob));
+  blob->ref = 1;
+  blob->data = parent->data + offset;
```

Comment:
void pointers don't really support pointer arithmetic, should be `(char*) parent->data + offset` to fix the windows build